security-analysis/src/main/java/com/powsybl/security/json/SecurityAnalysisParametersDeserializer.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2018, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.security.SecurityAnalysisParameters;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+class SecurityAnalysisParametersDeserializer extends StdDeserializer<SecurityAnalysisParameters> {
```

Comment:
As there is no inheritence between LoadFlowParameters and SecurityAnalysisParamaters, I'm not sure it's a good idea that SecurityAnalysisParametersJsonModule inherit from LoadFlowParameters. It should be a composition.